### ISSUE TITLE
caluate time is too long 

### ISSUE THREAD

**Issue by Toothsmile** [rpng/kalibr_allan#14]
I used this program just to caluate allan var  spend 30 mins, but cant work out ,i dont why need so long time

**Comment by tuandle** [comment]
@goldbattle Thanks for sharing the program. 
I have the same problem. I try to reproduce your `imu_visensor` result. I let the matlab program runs for 2 hours but it still doesn't finish. 
Any idea what might cause the problem or is it expected to be super heavy computational load?

**Comment by araujokth** [comment]
I am having the same issue. Is this expected? I have ran now for 1 hour and no result yet. I will leave this on during the night and see if anything happens. Also trying with the imu_visensor example. Tried both with matlab 2016b and 2019b and on ubuntu 16.04

UPDATE: I got a result after 2.5 hours. If anyone has some tips to speed this up it would be great. I have the feeling we are all having some wrong configuration somewhere that is making the computation very slow...
